bindings/matrix-sdk-ffi/src/session_verification.rs

```diff
@@ -208,6 +213,7 @@ impl SessionVerificationController {
                                 .iter()
                                 .map(|e| {
                                     Arc::new(SessionVerificationEmoji {
+                                        code: e.code.to_owned(),
```

Comment:
nit: `to_owned()` shouldn't be required here, because `e.code` can be copied at will by the compiler (it's an u32, which implements `Copy`, see other comment from Ivan).